xopt/generators/bayesian/base_model.py

```diff
@@ -34,14 +35,22 @@ def build_model(
         outcome_names: List[str],
         data: pd.DataFrame,
         input_bounds: Dict[str, List] = None,
+        dtype: torch.dtype = torch.double,
```

Comment:
the tkwargs should be an attribute OR an argument to the build model functions, not both

Reply:
Sorry, I don't understand what you mean. I think I've removed all attributes apart from the `use_cuda` flag in the generator. I had to add the `tkwargs` arguments to `build_model_from_vocs` as well though. That's why they appear twice.

Reply:
ah sorry, I thought these were added as pydantic attributes